kotlinx-coroutines-core/jvm/test/linearizability/LockFreeListLCStressTest.kt

```diff
@@ -5,16 +5,16 @@
 
 package kotlinx.coroutines.linearizability
 
-import com.devexperts.dxlab.lincheck.*
-import com.devexperts.dxlab.lincheck.annotations.*
-import com.devexperts.dxlab.lincheck.paramgen.*
-import com.devexperts.dxlab.lincheck.strategy.stress.*
 import kotlinx.coroutines.*
 import kotlinx.coroutines.internal.*
+import org.jetbrains.kotlinx.lincheck.annotations.*
+import org.jetbrains.kotlinx.lincheck.annotations.Operation
+import org.jetbrains.kotlinx.lincheck.paramgen.*
+import org.jetbrains.kotlinx.lincheck.verifier.*
 import kotlin.test.*
 
-@Param(name = "value", gen = IntGen::class, conf = "1:3")
-class LockFreeListLCStressTest : TestBase() {
+@Param(name = "value", gen = IntGen::class, conf = "1:5")
```

Comment:
This test should use a narrow range of values (`1:3`), because the key operation whose linearizability is checked here is `removeFirstOrPeekIfNotSame` which must have a high change of using the same value as the recently added nodes.